test/conformance/service_test.go

```diff
@@ -536,4 +536,47 @@ func TestReleaseService(t *testing.T) {
 	}
 }
 
+func TestMultipleNamespace(t *testing.T) {
+	t.Parallel()
+
+	altServiceNamespace := fmt.Sprintf("%s-%s", test.ServingNamespace, test.ObjectNameForTest(t))
+	serviceName := test.ObjectNameForTest(t)
+
+	defaultClients := e2e.Setup(t) // This one uses the default namespace `test.ServingNamespace`
+	altClients := e2e.SetupWithNamespace(t, altServiceNamespace)
+
+	_, err := test.CreateNamespace(altClients, altServiceNamespace)
```

Comment:
Q: but `defaultClients` would work here just as well right?

Reply:
Yup, it'll work fine to create the namespace since namespaces aren't namespace dependent.